src/lib/toolbar.ts

```diff
@@ -173,10 +173,15 @@ export const toolbarConfig: Record<ToolbarModeName, ToolbarMode> = {
         links: [{ label: 'KCL docs', url: 'https://zoo.dev/docs/kcl/fillet' }],
       },
       {
-        id: 'chamfer',
-        onClick: () => console.error('Chamfer not yet implemented'),
+        id: 'chamfer3d',
+        onClick: ({ commandBarSend }) =>
+          commandBarSend({
+            type: 'Find and select command',
+            data: { name: 'Chamfer', groupId: 'modeling' },
+          }),
         icon: 'chamfer3d',
-        status: 'kcl-only',
+        status: DEV || IS_NIGHTLY_OR_DEBUG ? 'available' : 'kcl-only',
+        disabled: (state) => !state.can({ type: 'Chamfer' }),
```

Comment:
```suggestion
```

I believe we've stopped using the `disabled` config option so that the toolbar buttons are always enabled, and their argument steps can just handle the validation, since ever toolbar button has some sort of selection argument first and that way we can let users just always try to use them.

Reply:
Ah good catch! @max-mrgrsk are you able to commit that?

Reply:
I think we can take care of this :) hitting the button